provision/gcp-instance/gcp-cors.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "origin": ["*"],
+    "responseHeader": ["Content-Type"],
+    "method": ["GET"],
+    "maxAgeSeconds": 3600
+  }
+]
```

Comment:
is this code related to certbot provisioning? Or is it a fix of something else?

Reply:
No, it's not. It's really part of the other PR and making storage buckets accessible to Timemap. Will remove it and add there.